OsuApiClient/Net/Requests/RequestHandler/DefaultRequestHandler.cs

```diff
@@ -21,7 +22,8 @@ internal sealed class DefaultRequestHandler(
     IOsuClientConfiguration configuration
 ) : IRequestHandler
 {
-    private readonly SemaphoreSlim _semaphore = new(1, 1);
+    private const string GlobalMutexName = "OTR_2b1c2e2f-9c62-4433-9066-dfb7a397d03d";
```

Comment:
```suggestion
    private const string GlobalMutexName = "Global\OTR_2b1c2e2f-9c62-4433-9066-dfb7a397d03d";
```

https://learn.microsoft.com/en-us/dotnet/api/system.threading.mutex.-ctor?view=net-9.0#system-threading-mutex-ctor(system-boolean-system-string)

> The name may be prefixed with Global\ or Local\ to specify a namespace. When the Global namespace is specified, the synchronization object may be shared with any processes on the system. When the Local namespace is specified, which is also the default when no namespace is specified, the synchronization object may be shared with processes in the same session.

I guess we need to use this convention for the name